frontend/lib/src/components/elements/PageLink/PageLink.tsx

```diff
@@ -95,7 +99,13 @@ function PageLink(props: Props): ReactElement {
             rel="noreferrer"
             onClick={handleClick}
           >
-            {element.icon && <EmojiIcon size="lg">{element.icon}</EmojiIcon>}
+            {element.icon && (
```

Comment:
On screenshot diffs material icon looks a little bit smaller, but I checked via CSS properties, in both cases it is `1.25 rem` , the size diff probably related to individual emoji / icon sizes